src/components/pages/my-bookings/MyBookingsLists/BookingsList/QrCode/QrCode.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import AbsoluteFooterContainer from '../../../../../layout/AbsoluteFooter/AbsoluteFooterContainer'
+
+const QrCode = props => {
```

Comment:
```suggestion
const QrCode = ({ humanizedBeginningDatetime, offerName, qrCode, token, venueName }) => {
```
Comme ça tu peux supprimer le `return`.

Reply:
👍